docker/network/functional/Dockerfile.centos-8

```diff
@@ -1,14 +1,20 @@
-FROM quay.io/ovirt/buildcontainer:stream8
+FROM centos/centos:stream8
```

Comment:
We used quay.io before - who provide this container? If this goes to dockerhub, we may have throttling issues.

Reply:
Right, let's put it back just to be safe